### PR TITLE
Add Release configuration file for Xcode.

### PR DESCRIPTION
**Pull request by neerajwahi**
This lets developers customize `DEVELOPMENT_TEAM` without pushing those changes.
Implements #68.

### CODE REVIEW

File: ios/ZulipMobile/Release.xcconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//
+//  Release.xcconfig
+//  ZulipMobile
+//
+//  This file is in .gitignore. Add your own key to DEVELOPMENT_TEAM
+
+CODE_SIGN_IDENTITY = iPhone Developer
+DEVELOPMENT_TEAM =
```
Comment (timabbott) [inline ios/ZulipMobile/Release.xcconfig]: Is this supposed to be here?  Maybe should be named `Release.xcconfig.example`?

Comment (neerajwahi) [inline ios/ZulipMobile/Release.xcconfig]: Yeah that's a good idea. I was looking for a way to check it in but then have git ignore future changes to it but doesn't seem like that behavior is really possible

Comment (timabbott) [inline ios/ZulipMobile/Release.xcconfig]: yeah, it's not.

Comment (timabbott) [inline ios/ZulipMobile/Release.xcconfig]: One could probably put a provisioning script in place that copies from the example file, but I don't think it's likely to be worth it...

**Pull request closed without merging**